src/app/voting/add-edit/add-edit.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-add-edit',
+  templateUrl: './add-edit.component.html',
+  styleUrls: ['./add-edit.component.css'],
```

Comment:
```suggestion
  styleUrls: ['./add-edit.component.css', '../../program/shared/event-dialog/event-dialog.component.css'],
```
Reference CSS files.